models/articles.model.js

```diff
@@ -0,0 +1,12 @@
+const db = require("../db/connection");
+
+exports.selectArticleById = (article_id) => {
+  return db
+    .query(`SELECT * FROM articles WHERE article_id = $1;`, [article_id])
+    .then(({ rows }) => {
+      if (!rows.length) {
+        return Promise.reject({ status: 404, msg: "Not Found" });
+      }
+      return rows;
```

Comment:
At this point, the requested article is the first and only item in the rows array. We should make sure that we just return the {} and not the [{}] here. You will also need to update your test to reflect this :)